### PR TITLE
wip: Fix DownloadOnlyDocker test on github actions

### LINKED ISSUE #6869: Fix DownloadOnlyDocker test on github actions

### PR DESCRIPTION
**Pull request by priyawadhwa**
Fixes #6869

### CODE REVIEW

**Comment by priyawadhwa** [comment]
It seems this test is still failing on github actions because the docker daemon is showing the digest as none type:

```
 aaa_download_only_test.go:205: expected image does not exist in local daemon; got gcr.io/k8s-minikube/kicbase:v0.0.7@<none>
        node:10-alpine@sha256:e8d05985dd93c380a83da00d676b081dad9cce148cb4ecdf26ed684fcff1449c
        node:10@sha256:cfc462a03d170c09332f28cb358d796575bc0ac2e2b47d97e10810f6c4acdd2e
        node:12-alpine@sha256:bba77d0ca8820b43af898b3c50d4e8b68dc703ebbd958319af2f21f2d3c309f5
        node:12@sha256:c185973d2d8d60d2f99abe1285e1ad0887a42f313f3befa6ae8f2429304ee09c
        buildpack-deps:stretch@sha256:8bcd320ec29cf67052985f28891586fb853051f69ad0646fc7a49f47d6e3ee1a
        debian:9@sha256:da5274336981301e2c5f2edb54eaa4dccee70c39506f96d39377b46ea75e804e
        debian:8@sha256:0e7a709bdf94e15391e9658271197e28a8c993f4d6bb8ee49d370886a9893351
        alpine:3.8@sha256:2bb501e6173d9d006e56de5bce2720eb06396803300fe1687b58a7ff32bf4c14
        alpine:3.9@sha256:115731bab0862031b44766733890091c17924f9b7781b79997f5f163be262178
        alpine:3.10@sha256:7c3773f7bcc969f03f8f653910001d99a9d324b4b9caa008846ad2c3089f5a5f
        jekyll/builder:latest@sha256:875de39aa06300c749e26a79f39cdfa31404e4f2dbd2953cd4ebb0442bffda9e
        alpine:3.7@sha256:8421d9a84432575381bfabd248f1eb56f3aa21d9d7cd2511583c68c9b7511d10
        mcr.microsoft.com/azure-pipelines/node8-typescript:latest@sha256:e52e60b9f71183969830a3664279b5d8c799b4b0ec2c25a0686f7c02f6a9669a
         wanted gcr.io/k8s-minikube/kicbase:v0.0.7@sha256:a6f288de0e5863cdeab711fa6bafa38ee7d8d285ca14216ecf84fcfb07c7d176

```

**Comment by medyagh** [comment]
> /ok-to-test

for github actions we dont need to say ok-to-test it runs on all PRs.
ok-to-test is only to invoke jenkins for additional testing,

I recommend for this PR, just only invoke github action tests

File: test/integration/aaa_download_only_test.go [hunk 1]
```diff
@@ -199,18 +197,19 @@ func TestDownloadOnlyDocker(t *testing.T) {
...
 	}
-	if _, err := daemon.Image(ref); err != nil {
-		t.Errorf("expected image does not exist in local daemon: %v", err)
+	want := kic.BaseImage
+	if !strings.Contains(string(images), want) {
+		t.Errorf("expected image does not exist in local daemon; got:\n%s wanted:\n%s", string(images), want)
 	}
 }
 
 func runningDockerDriver(startArgs []string) bool {
```
Comment (medyagh) [inline test/integration/aaa_download_only_test.go]: please move this to main.go 

and name it DockerDriver()
same as https://github.com/kubernetes/minikube/blob/82900a7811e479755e03ae020ebb44152dfd93d8/test/integration/main.go#L63
```
// NoneDriver returns whether or not this test is using the none driver
func NoneDriver() bool {
	return strings.Contains(*startArgs, "--driver=none")
}
```

**Comment by priyawadhwa** [comment]
Closing for now until I get back to this

**Pull request closed without merging**